features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Login/Login.js

```diff
@@ -81,8 +81,10 @@ class Login extends Component {
 
         let loginPromise = this.authManager.authenticateUser(username, password, detailedValue);
         loginPromise.then((response) => {
+            debugger;
```

Comment:
Let's remove these debuggers from the code.

Reply:
done